### PR TITLE
User login

### PR DESCRIPTION
**Pull request by DanielFleace1**
Created userLogin Page. Created inputs for username ( or email depending on what is decided) and password. Provided a timed alert  if incorrect password is provided. Added NavBar to bottom of page.

### CODE REVIEW

**Comment by rmcguire6** [comment]
Dan you need to change this pull request to merge to dev branch. We only merge dev to main.

**Comment by DanielFleace1** [comment]
@rmcguire6 meant to do that ! Thanks.

File: src/pages/Login.tsx [hunk 1]
```diff
@@ -1,15 +1,86 @@
...
+    const [incorrectPassword, setIncorrectPassword] = React.useState<boolean>(false);
+    
+    const handleSubmit = (e: React.FormEvent<HTMLFormElement>) => {
+        e.preventDefault()
+        const target = e.target as typeof e.target & {
+            username: { value: string };
+            password: { value: string };
+          };
+
+        //if(target correct ) { redirect to logged in home page}
```
Comment (ArturoGrullon) [inline src/pages/Login.tsx]: These comments won't be needed in development

Comment (ArturoGrullon) [reply src/pages/Login.tsx]: I think I see what are you are doing, this is for the future, right? 

A good  `//Todo: handle a wrong password` is good enough, in my opinion

File: src/pages/Login.tsx [hunk 2]
```diff
@@ -1,15 +1,86 @@
...
+                    value={password}
+                    onChange={(e) => handleInputChange(e)}
+                />
+                </label>
+                <br/>
+                <input type="submit"  />
+            </form>
+            <NavBar />
+            {
+                incorrectPassword ? <div> Incorrect Username or Password!</div>  :
```
Comment (ArturoGrullon) [inline src/pages/Login.tsx]: If this is meant to only show something when it's true you might prefer 

`{incorrectPaswword && <div> Incorrect...</div>}`

**Review by ArturoGrullon — APPROVED** [review]
Overall is great, and taking into consideration you haven't used either git and typescript before is quite impressive. 

Great job 🙂

File: src/pages/Login.tsx (hunk 1 above)
Comment (DanielFleace1) [reply src/pages/Login.tsx]: Thanks. Yes it is for the future. For brevity I changed it and re-pushed using your recommendations !

File: src/pages/Login.tsx (hunk 2 above)
Comment (DanielFleace1) [reply src/pages/Login.tsx]: Thanks! That is a little cleaner. Changed and re-pushed.

**Comment by DanielFleace1** [comment]
Thanks @ArturoDucasse! It helped to have example components to look at from the rest of the team!

**Review by JessieFrance — APPROVED** [review]
I think this looks good to merge. Great work Dan!

**Comment by DanielFleace1** [comment]
@JessieFrance Thanks for the review !

**Pull request merged**